### ISSUE TITLE
is_dir() not accessing ContentPath?

### ISSUE THREAD

**Issue by kripper** [peachpiecompiler/peachpie#1080]
In a template Web App project, in `Program.cs` we use `ResolveContentPath()` to tell Peachpie that the content files should be located insde `../Website/`  [1].
But if we evaluate is_dir('here'), ResolveContentPath() is not used, so the files are checked in `Server/here` insteead of `../Website/here`

[1] Program.cs:
```
        static string ResolveContentPath()
        {
            var devcontent = Path.GetFullPath("../Website");
            if (Directory.Exists(devcontent))
            {
                return devcontent;
            }

            return Directory.GetCurrentDirectory();
        }
```

**Comment by jakubmisek** [comment]
we're reporting everything to Debug Output window in .NET

**Comment by kripper** [comment]
Right. And I believe this also goes all to the logs even on production which is a problem when you have millions of unwanted notices and you don't want to spend months adapting legacy code (specially when it's not yours) or when having undefined variables is your coding style choice and the reason you love PHP.

BTW, there is a flame war in the PHP community regarding the imposed coding style since version 8:

About 40% of the developers considered it a bad decision, very disrespectful and/or that PHP's simplicity was destroyed. As a workaround, some developers suggested 1) to disable logging notices using `error_reporting(E_ALL ^ E_NOTICE ^ E_DEPRECATED ^ E_STRICT)` and 2) to create your own error handler and ignore specific warnings that were previously handled as notices.

This workaround is not working in Peachpie, becauses 1) `error_reporting` didn't affect the logs and 2) it is ignoring the value returned by the error handler function, where returning `true` means "ignore the error completely and don't even log it".

My suggestion is to implement both to avoid forks, because even when 60% is bigger than 40%, that's not a reason to crash them (force them to rewrite their legacy code instead of simply solve it with a single configuration option).
